tools/topology/topology2/include/dais/ssp.conf

```diff
@@ -91,9 +91,17 @@ Class.Dai."SSP" {
 		constraints {
 			!valid_values [
 				"lbm_mode"
+				"bt_sideband"
+				"render_feedback"
+				"lbm_mode,bt_sideband"
+				"lbm_mode,render_feedback"
```

Comment:
this doesn't scale, we don't want to create a new definition for each possible combination. This needs to be a mask or something.

Reply:
I know this change looks stupid... Another way is to add new property like 'nhlt_quirk' to handle this.

Reply:
even if we add a new property, what we have today is 'global' and has no notion of direction, but the NHLT tags are not-only endpoint specific but direction specific. I am not sure how to reconcile the two, we have no placeholder for direction-specific information.

Reply:
I think here we could have the NHLT object and only assign direction to it when we instanciate it ? @ranj063 ?